packages/sdk/src/across/gasFeeCalculator.ts

```diff
@@ -23,7 +25,20 @@ export default async (
   gas: number,
   tokenAddress?: string
 ): Promise<ReturnType> => {
-  const gasPrice = await ethersProvider.getGasPrice();
+  const latestBlockNumber = await (ethersProvider as ethers.providers.JsonRpcProvider).getBlockNumber();
+  const { baseFeePerGas } = await (ethersProvider as ethers.providers.JsonRpcProvider).getBlock(latestBlockNumber);
```

Comment:
You can usually do `getBlock("latest")`, might save a call

Reply:
thx